docs/registry/create/_index.md

```diff
@@ -685,17 +690,62 @@ Make sure to set up a Python virtual environment in the directory your module is
 
     # Be sure to use `exec` so that termination signals reach the python process,
     # or handle forwarding termination signals manually
-    exec $PYTHON <your-src-dir-if-inside>/main.py $@
-   ```
+    exec $PYTHON -m src.main $@
+```
```

Comment:
I think it's this 4-whitespace indented `sh` code fence that is causing the render issues on the rest of this page. When I dedent this whole code fence to be in line with the rest of the surrounding copy, the expected formatting after the double-tabsets returns! Huzzah! 🎉